test-suite/misc/attributes.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -e
+
+export COQBIN=$BIN
+export PATH=$COQBIN:$PATH
+
+cd misc/attributes/
+
+rocq makefile -f _CoqProject -o Makefile
+
+make clean
```

Comment:
might be nicer to make a fresh subdir and copy sources into it instead of make clean
```suggestion
rm -rf _test
mkdir _test
find . -maxdepth 1 -not -name . -not -name _test -exec cp -r '{}' -t _test ';'
cd _test

rocq makefile -f _CoqProject -o Makefile
```
also add a .gitignore containing just `_test` in test-suite/misc/attributes